RecoMET/METFilters/python/ecalBadCalibFilter_cfi.py

```diff
@@ -29,4 +29,39 @@
                                                        872423215,872416066,872435036,872439336,
                                                        872420273,872436907,872420147,872439731,
                                                        872436657,872420397,872439732,872439339,
-                                                       872439603,872422436])
+                                                       872439603,872422436,872439861,872437051,
+                                                       872437052,872420649,872422436,872421950,
+                                                       872437185,872422564,872421566,872421695,
+                                                       872421955,872421567,872437184,872421951,
+                                                       872421694,872437056,872437057,872437313])
+
+
+ecalBadCalibReducedMINIAODFilter = cms.EDFilter(
```

Comment:
I would suggest giving this a more peculiar name, as there is no a-priori reason to define bad crystals in a different way when running on AOD or MINIAOD.
I would name it ecalBadCalibReducedMINIAODPre104XFilter to make it clear it has to be run and considered only when running on MINIAOD produced in older releases.